cmd/kube-dns/app/options/options.go

```diff
@@ -187,4 +189,6 @@ func (s *KubeDNSConfig) AddFlags(fs *pflag.FlagSet) {
 			"used in conjunction with federations or config-map flag.")
 	fs.DurationVar(&s.ConfigPeriod, "config-period", s.ConfigPeriod,
 		"period at which to check for updates in config-dir.")
+	fs.BoolVar(&s.EnableProfiling, "profiling", s.EnableProfiling, "If true enables profiling via web interface on /debug/pprof handler.")
```

Comment:
probably don't need a flag as this will be bound on localhost, we can just have this always be available.